pyInterface/bindings/storageFormats/eventMetadata_py.cc

```diff
@@ -86,6 +86,9 @@ void rpwa::py::exportEventMetadata() {
 		.def("productionKinematicsParticleNames", &eventMetadata_productionKinematicsParticleNames)
 		.def("decayKinematicsParticleNames", &eventMetadata_decayKinematicsParticleNames)
 		.def("additionalSavedVariableLables", &eventMetadata_additionalSavedVariableLables)
```

Comment:
Not your fault, but it would be nice to fix typo: "additionalSavedVariableLables" -> "additionalSavedVariableLabels"